src/rt-comparison-chart/RtComparisonChart.tsx

```diff
@@ -261,10 +261,12 @@ export const isRtComparisonData = (
 const RtComparisonChart: React.FC<{ data: RtComparisonData[] }> = ({
   data,
 }) => {
-  const maxExtent = Math.ceil(
+  const maxRtHigh90 = Math.ceil(
     Math.max(...data.map((record) => record.values.high90 || 0)),
   );
 
+  const maxExtent = maxRtHigh90 >= 4 ? maxRtHigh90 + 1 : 5;
```

Comment:
why not just `Math.max(maxRtHigh0 + 1, 5)`?

Reply:
Ah yes that much better, I'll update this soon.